content/millennium/r4/scheduling/appointment.md

```diff
@@ -268,6 +268,34 @@ The `ETag` response header indicates the current `If-Match` version to use on su
 
 <%= disclaimer %>
 
+### Example - Update cancelationReason
```

Comment:
This is adding a cancelationReason.

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/pull/593/commits/461871f87929b844e8cd3c9b03edf5f15ff3de0f#diff-af2fad7ff605afe25e51152d541f652713df8ea51a020d1eae9bc557c158d2cbR271